pkg/app/run.go

```diff
@@ -74,6 +74,11 @@ var CommandCreate = &cli.Command{
 			Usage: "Assign the host address for environment ssh acesss server listening",
 			Value: envd.Localhost,
 		},
+		&cli.IntFlag{
+			Name:  "shm-size",
+			Usage: "Configure the shared memory size (megabyte)",
+			Value: 0,
```

Comment:
I think we should set a larger default value, WDYT?

Reply:
How much? It also consumes memory. The default resource limit is 2GB of memory. 
- https://github.com/tensorchord/envd-server/blob/f9e9d2689a22edda828be867dde626eb188c6824/pkg/runtime/kubernetes/environment_create.go#LL176C30-L176C30

Reply:
Be default, the container will have 64MB of shared memory.

Reply:
I think for docker, we can keep 2GB memory. WDYT

Reply:
SGTM